rebasehelper/application.py

```diff
@@ -89,6 +90,8 @@ def __init__(self, cli_conf, execution_dir, results_dir, debug_log_file):
 
         self.kwargs['non_interactive'] = self.conf.non_interactive
 
+        self.kwargs['changelog-entry'] = self.conf.changelog_entry
```

Comment:
Replace hyphen with underscore. Hyphen is not allowed in identifier names so this can't be used e.g. as function argument.